pydantic/validators.py

```diff
@@ -258,6 +266,8 @@ def find_validators(type_, arbitrary_types_allowed=False):
         return []
     if type_ is Pattern:
         return pattern_validators
+    if getattr(type_, '__origin__', None) is collections.abc.Callable:
```

Comment:
this works for `Callable[[foo, bar], spam]` but not for just `Callable`. just simply `Callable`, eg.
```py
class MyModel(BaseModel):
    func: Callable
```

I think you need `type_ is Callable or getattr(type_, '__origin__', None) is Callable`

Reply:
I add a test to prove we work both with `Callable` and `Callable[[int], int]`.